Realm/Tests/PerformanceTests.m

```diff
@@ -29,7 +29,7 @@ + (RLMPropertyAttributes)attributesForProperty:(__unused NSString *)propertyName
 }
 @end
 
-#if !DEBUG
+//#if !DEBUG
```

Comment:
The `#if !DEBUG` makes them not show up in the Xcode UI, and this is rebased on a WIP branch that I need to be able to test.